### PR TITLE
Updating README for annotations and adding french translations

### PR DESCRIPTION
**Pull request by AxelDeneu**
See #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,37 @@ brille24_customer_options:
     resource: "@Brille24SyliusCustomerOptionsPlugin/Resources/config/app/routing.yml"
 ```
 
+* [Optional] If you're using annotations in the entities definition. Please inherit the class `Product` of the SyliusCustomerOptionsPlugin's `Product` class and the class `OrderItem` of the SyliusCustomerOptionsPlugin's `OrderItem` class:
```
Comment (mamazu) [inline README.md]: Could you remove this section. We already added this in a different commit.

File: src/Resources/translations/messages.fr.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                message: Message
+            error_message: Message d'erreur
+            buttons:
+                add_condition: Ajouter une condition
+                delete_condition: Supprimer la condition
+                add_constraint: Ajouter une contrainte
+                delete_constraint: Supprimer la contrainte
+
+    grid:
+        options_remaining: "{0} |]0,Inf]and %count% more"
```
Comment (mamazu) [inline src/Resources/translations/messages.fr.yml]: Would that be
`{0} |]0,Inf]et %count% plus`

**Comment by AxelDeneu** [comment]
Branch modified to match recommandations :)

**Comment by mamazu** [comment]
Thanks for your contribution.

**Pull request merged**